INFO:root:dupa
INFO:root:choices
INFO:root:choices
ERROR:django.request:Internal Server Error: /addQuestion/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 45, in addQuestion
    return render(request, 'Recruiter/questions/addQuestion.html', {'question_form': question_form, 'category_form': category_form})
UnboundLocalError: local variable 'question_form' referenced before assignment
ERROR:django.request:Internal Server Error: /addQuestion/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 30, in addQuestion
    if question_form.is_valid() and category_form.is_valid():
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\forms.py", line 183, in is_valid
    return self.is_bound and not self.errors
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\forms.py", line 175, in errors
    self.full_clean()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\forms.py", line 386, in full_clean
    self._post_clean()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 413, in _post_clean
    self.instance.full_clean(exclude=exclude, validate_unique=False)
TypeError: full_clean() missing 1 required positional argument: 'self'
ERROR:django.request:Internal Server Error: /addQuestion/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 30, in addQuestion
    if question_form.is_valid() and category_form.is_valid():
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\forms.py", line 183, in is_valid
    return self.is_bound and not self.errors
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\forms.py", line 175, in errors
    self.full_clean()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\forms.py", line 386, in full_clean
    self._post_clean()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 413, in _post_clean
    self.instance.full_clean(exclude=exclude, validate_unique=False)
TypeError: full_clean() missing 1 required positional argument: 'self'
ERROR:django.request:Internal Server Error: /addQuestion/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: table Recruiter_question has no column named category_type_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 35, in addQuestion
    author=request.user
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 838, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 924, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 963, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1107, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: table Recruiter_question has no column named category_type_id
ERROR:django.request:Internal Server Error: /admin/Recruiter/question/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such column: Recruiter_question.category_type_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 1662, in changelist_view
    selection_note=_('0 of %(cnt)s selected') % {'cnt': len(cl.result_list)},
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 894, in execute_sql
    raise original_exception
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 884, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such column: Recruiter_question.category_type_id
ERROR:django.request:Internal Server Error: /admin/Recruiter/question/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such column: Recruiter_question.category_type_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 1662, in changelist_view
    selection_note=_('0 of %(cnt)s selected') % {'cnt': len(cl.result_list)},
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 894, in execute_sql
    raise original_exception
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 884, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such column: Recruiter_question.category_type_id
ERROR:django.request:Internal Server Error: /addQuestion/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.category_type_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 35, in addQuestion
    author=request.user
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 838, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 924, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 963, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1107, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.category_type_id
WARNING:django.request:Not Found: /favicon.ico
ERROR:root:summary
ERROR:django.request:Internal Server Error: /questions/6/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 23, in detail
    return render(request, 'Recruiter/questions/questionDetail.html', {'question': question})
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 458, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\urls\base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\urls\resolvers.py", line 497, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'editQuestion' not found. 'editQuestion' is not a valid view function or pattern name.
ERROR:django.request:Internal Server Error: /questions/6/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 23, in detail
    return render(request, 'Recruiter/questions/questionDetail.html', {'question': question})
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 458, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\urls\base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\urls\resolvers.py", line 497, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'editQuestion' not found. 'editQuestion' is not a valid view function or pattern name.
ERROR:django.request:Internal Server Error: /admin/Recruiter/question/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 1662, in changelist_view
    selection_note=_('0 of %(cnt)s selected') % {'cnt': len(cl.result_list)},
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 62, in __iter__
    for row in compiler.results_iter(results):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 842, in results_iter
    row = self.apply_converters(row, converters)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 827, in apply_converters
    value = converter(value, expression, self.connection, self.query.context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\operations.py", line 219, in convert_datetimefield_value
    value = parse_datetime(value)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\dateparse.py", line 94, in parse_datetime
    match = datetime_re.match(value)
TypeError: expected string or bytes-like object
ERROR:django.request:Internal Server Error: /admin/Recruiter/question/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 1662, in changelist_view
    selection_note=_('0 of %(cnt)s selected') % {'cnt': len(cl.result_list)},
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 62, in __iter__
    for row in compiler.results_iter(results):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 842, in results_iter
    row = self.apply_converters(row, converters)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 827, in apply_converters
    value = converter(value, expression, self.connection, self.query.context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\operations.py", line 219, in convert_datetimefield_value
    value = parse_datetime(value)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\dateparse.py", line 94, in parse_datetime
    match = datetime_re.match(value)
TypeError: expected string or bytes-like object
ERROR:django.request:Internal Server Error: /admin/Recruiter/question/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 1662, in changelist_view
    selection_note=_('0 of %(cnt)s selected') % {'cnt': len(cl.result_list)},
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 62, in __iter__
    for row in compiler.results_iter(results):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 842, in results_iter
    row = self.apply_converters(row, converters)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 827, in apply_converters
    value = converter(value, expression, self.connection, self.query.context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\operations.py", line 219, in convert_datetimefield_value
    value = parse_datetime(value)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\dateparse.py", line 94, in parse_datetime
    match = datetime_re.match(value)
TypeError: expected string or bytes-like object
ERROR:django.request:Internal Server Error: /questions/edit/6/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 48, in editQuestion
    question = get_object_or_404(Question, pk=questionId)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\shortcuts.py", line 85, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 374, in get
    num = len(clone)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 62, in __iter__
    for row in compiler.results_iter(results):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 842, in results_iter
    row = self.apply_converters(row, converters)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 827, in apply_converters
    value = converter(value, expression, self.connection, self.query.context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\operations.py", line 219, in convert_datetimefield_value
    value = parse_datetime(value)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\dateparse.py", line 94, in parse_datetime
    match = datetime_re.match(value)
TypeError: expected string or bytes-like object
ERROR:django.request:Internal Server Error: /admin/Recruiter/question/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\admin\options.py", line 1662, in changelist_view
    selection_note=_('0 of %(cnt)s selected') % {'cnt': len(cl.result_list)},
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 62, in __iter__
    for row in compiler.results_iter(results):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 842, in results_iter
    row = self.apply_converters(row, converters)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 827, in apply_converters
    value = converter(value, expression, self.connection, self.query.context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\operations.py", line 219, in convert_datetimefield_value
    value = parse_datetime(value)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\dateparse.py", line 94, in parse_datetime
    match = datetime_re.match(value)
TypeError: expected string or bytes-like object
ERROR:django.request:Internal Server Error: /questions/edit/6/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 61, in editQuestion
    return render(request, 'Recruiter/questions/editQuestion.html', {'form': form})
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 204, in render
    template = context.template.engine.get_template(template_name)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\engine.py", line 162, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\engine.py", line 136, in find_template
    name, template_dirs=dirs, skip=skip,
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loaders\base.py", line 44, in get_template
    contents, origin, origin.template_name, self.engine,
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 191, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 230, in compile_nodelist
    return parser.parse()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 515, in parse
    raise self.error(token, e)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 513, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 827, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 515, in parse
    raise self.error(token, e)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 513, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 983, in do_if
    nodelist = parser.parse(('endif',))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 515, in parse
    raise self.error(token, e)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 513, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\widget_tweaks\templatetags\widget_tweaks.py", line 155, in render_field
    attr, sign, value = dct['attr'], dct['sign'], parser.compile_filter(dct['value'])
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 694, in __init__
    filter_func = parser.find_filter(filter_name)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 606, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'input_value'
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 22, in _wrapped_view
    if test_func(request.user):
AttributeError: 'QuestionView' object has no attribute 'user'
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 22, in _wrapped_view
    if test_func(request.user):
AttributeError: 'QuestionView' object has no attribute 'user'
ERROR:django.request:Internal Server Error: /questions/edit/6/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
TypeError: as_view() takes 1 positional argument but 2 were given
ERROR:django.request:Internal Server Error: /questions/edit/6/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
TypeError: as_view() takes 1 positional argument but 2 were given
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 12, in form_valid
    success_message = self.get_success_message(form.cleaned_data)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\QuestionView.py", line 21, in get_success_message
    calculated_field=self.object.calculated_field,
AttributeError: 'Question' object has no attribute 'calculated_field'
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: 'localhost:8000'. You may need to add 'localhost' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
WARNING:django.request:Not Found: /Recruiter/
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /Recruiter/
WARNING:django.request:Not Found: /Recruiter/
WARNING:django.request:Not Found: /Recruiter/
ERROR:django.request:Internal Server Error: /questions/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 17, in questions
    user_list = models.User.objects.all()
AttributeError: module 'django.db.models' has no attribute 'User'
ERROR:django.request:Internal Server Error: /questions/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 17, in questions
    user_list = models.auth.User.objects.all()
AttributeError: module 'django.db.models' has no attribute 'auth'
ERROR:django.request:Internal Server Error: /questions/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 20, in questions
    return render(request, 'Recruiter/questions/questions.html', context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loaders\base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 198, in compile_nodelist
    return parser.parse()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 956, in do_if
    nodelist = parser.parse(('elif', 'else', 'endif'))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 805, in do_for
    " 'for x in y': %s" % token.contents)
django.template.exceptions.TemplateSyntaxError: 'for' statements should use the format 'for x in y': for question, user in questionList, filter.qs
WARNING:django.security.csrf:Forbidden (CSRF token missing or incorrect.): /login/
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 468, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 838, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 924, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 963, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1112, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 468, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 838, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 924, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 963, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1112, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 468, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 838, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 924, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 963, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1112, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\Recruiter\views\views.py", line 11, in home
    return render(request, 'Recruiter/home.html')
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 208, in render
    return self._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 994, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 994, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\loader_tags.py", line 70, in render
    result = block.nodelist.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 994, in render
    bit = node.render_annotated(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\template\defaulttags.py", line 439, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\urls\base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\urls\resolvers.py", line 392, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns)
django.urls.exceptions.NoReverseMatch: Reverse for 'logout' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 42, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 453, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 908, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 947, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1045, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1054, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: 'localhost:8000'. You may need to add 'localhost' to ALLOWED_HOSTS.
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: 'localhost:8000'. You may need to add 'localhost' to ALLOWED_HOSTS.
WARNING:django.request:Not Found: /Recruiter/
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 451, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 806, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 836, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 922, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 961, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1063, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1099, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 451, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 806, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 836, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 922, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 961, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1063, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1099, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 451, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 806, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 836, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 922, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 961, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1063, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1099, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
ERROR:django.request:Internal Server Error: /questions/add/
Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 217, in post
    return super(BaseCreateView, self).post(request, *args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 183, in post
    return self.form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\contrib\messages\views.py", line 11, in form_valid
    response = super(SuccessMessageMixin, self).form_valid(form)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\views\generic\edit.py", line 162, in form_valid
    self.object = form.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\forms\models.py", line 451, in save
    self.instance.save()
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 806, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 836, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 922, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\base.py", line 961, in _do_insert
    using=using, raw=raw)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\query.py", line 1063, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1099, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\utils\six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Users\music\Documents\GitHub\RecruiterSite\venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: Recruiter_question.author_id
